Wrappers/Python/cil/plugins/tigre/ProjectionOperator.py

```diff
@@ -76,34 +134,45 @@ def __init__(self, image_geometry=None, acquisition_geometry=None, direct_method
 
         if acquisition_geometry_old is not None:
             acquisition_geometry = acquisition_geometry_old
-            logging.warning("aquisition_geometry has been deprecated. Please use acquisition_geometry instead.")
+            logging.warning(
+                "aquisition_geometry has been deprecated. Please use acquisition_geometry instead."
+            )
 
         if acquisition_geometry is None:
-            raise TypeError("Please specify an acquisition_geometry to configure this operator")
+            raise TypeError(
+                "Please specify an acquisition_geometry to configure this operator"
+            )
 
         if image_geometry == None:
             image_geometry = acquisition_geometry.get_ImageGeometry()
 
         device = kwargs.get('device', 'gpu')
         if device != 'gpu':
-            raise ValueError("TIGRE projectors are GPU only. Got device = {}".format(device))
+            raise ValueError(
```

Comment:
splitting this on to 3 lines feels excessive... but ok

Reply:
probably autoyapf